### PR TITLE
Update dotnet-contribute.md

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: Contribute/dotnet/dotnet-contribute.md [hunk 1]
```diff
@@ -94,6 +94,9 @@ docs
             Program.vb
 ```
 
+> [!NOTE]
+> The language folders under snippets are not needed in the language guide area, where only one language is assumed.
```
Comment (Youssef1313) [inline Contribute/dotnet/dotnet-contribute.md:98]: This is related to https://github.com/dotnet/docs/issues/19791#issuecomment-668072235

File: Contribute/dotnet/dotnet-contribute.md [hunk 2]
```diff
@@ -149,13 +152,12 @@ Samples are full programs and libraries meant for download. They may be small in
 3. Your sample should include **appropriate exception handling**. It should handle all exceptions that are likely to be thrown in the context of the sample. For example, a sample that calls the [Console.ReadLine](https://docs.microsoft.com/dotnet/api/system.console.readline) method to retrieve user input should use appropriate exception handling when the input string is passed as an argument to a method. Similarly, if your sample expects a method call to fail, the resulting exception must be handled. Always handle the specific exceptions thrown by the method, rather than base class exceptions such as [Exception](https://docs.microsoft.com/dotnet/api/system.exception) or [SystemException](https://docs.microsoft.com/dotnet/api/system.systemexception).
 
 4. If your sample builds a standalone package, you must include the runtimes used by our CI build system, in addition to any runtimes used by your sample:
+
     - `win7-x64`
     - `win8-x64`
     - `win81-x64`
     - `ubuntu.16.04-x64`
 
-We will have a CI system in place to build these projects shortly.
```
Comment (Youssef1313) [inline Contribute/dotnet/dotnet-contribute.md:157]: @adegeo, I deleted this sentence as I think it's outdated now. Can you confirm if you want to delete it?

Comment (adegeo) [reply Contribute/dotnet/dotnet-contribute.md:157]: I think the runtime list above also needs to change to win10-x64 only as our CI system is currently only that.

Comment (Youssef1313) [reply Contribute/dotnet/dotnet-contribute.md:157]: @adegeo I find the whole sentence isn't clear to me a bit.

> 4. If your sample builds a standalone package, you must include the runtimes used by our CI build system, in addition to any runtimes used by your sample:
>
>    - `win7-x64`
>    - `win8-x64`
>    - `win81-x64`
>    - `ubuntu.16.04-x64`

If the list needs to be changed, I think it's worth to make the sentence more clear. Do you have any suggestions?

Comment (tdykstra) [reply Contribute/dotnet/dotnet-contribute.md:157]: > it's worth to make the sentence more clear. 

I agree

Comment (adegeo) [reply Contribute/dotnet/dotnet-contribute.md:157]: I'm not exactly sure what t is trying to communicate. We should find the original author

Comment (Jim-Parker) [reply Contribute/dotnet/dotnet-contribute.md:157]: @BillWagner can you help clarify?

Comment (BillWagner) [reply Contribute/dotnet/dotnet-contribute.md:157]: To match what we do now, we should remove the entire list item (4 in the article). We don't build on multiple platforms, so we don't need to specify which platforms to build packages for.

**Comment by Jim-Parker** [comment]
@Youssef1313 is this ready to merge now?

**Comment by Youssef1313** [comment]
@Jim-Parker Yes I think.

**Comment by adegeo** [comment]
@jim-parker go for it!

**Pull request merged**